tests/e2e/tests/content-manager/preview.spec.ts

```diff
@@ -104,7 +105,7 @@ describeOnCondition(edition === 'EE')('Preview', () => {
     // Navigate to the published tab
     await clickAndWait(page, page.getByRole('tab', { name: /^Published$/ }));
 
-    const updatedIframe = await page.getByTitle('Preview');
+    const updatedIframe = page.getByTitle('Preview');
     const srcPublished = await updatedIframe.getAttribute('src');
```

Comment:
I'm a little confused by what is async and what is not in the playwright API. Is it clear for you? Are you confident in these changes because missing an await can very easily lead to flaky tests

Reply:
`page.getByTitle` does not return a promise, so I'm confident it wasn't useful, vscode was warning me about it.

But you're right that we should await an expect here

Reply:
Oh I think I get it. It looks like getting a locator is not async but the actions on that locator are async. I think there are many places where we await when we don't need. But does that have any sort of real impact 🤔 or just ugly?

I did just notice getAttribute is discouraged, could you please try what they recommend with `toHaveAttribute` to avoid a flaky test 🙏 :
<img width="1100" alt="Screenshot 2024-12-02 at 17 25 32" src="https://github.com/user-attachments/assets/d6966a12-9d82-4d3b-87ed-af2b3a2dc696">

Reply:
> It looks like getting a locator is not async but the actions on that locator are async

that's my understanding too. But not actions return promises. For example I tried adding a step to make sure it's there but it also doesn't return a promise, so not useful:

![screenshot 2024-12-02 at 17 24 30@2x](https://github.com/user-attachments/assets/f074d2ba-2707-4ad1-ab9f-343d7b632728)

Extra awaits don't create issues except they make me sad when I see warnings in my editor, so I like to 🧹🧹🧹 when I find them

Reply:
Yes I agree good to clean them and only use them where needed, thanks for doing that! I don't get those errors though I wonder why 😞